package.json

```diff
@@ -37,8 +37,8 @@
 	"activationEvents": [
 		"onLanguage:mongo",
 		"onView:cosmosDBExplorer",
-		"onCommand:cosmosDB.executeMongoCommand",
-		"onCommand:cosmosDB.executeAllMongoCommands",
+		"onCommand:cosmosDB.api.getConnectionString",
```

Comment:
So I believe we were going to discuss whether it would be better to provide an API off of the extension object rather than using a command ID, right?  Were you planning on merging before that discussion?

Reply:
Right, as I understand Eric doesn't want to provide any CosmosDB objects in AppService. I can continue to work on appservice changes even without merging, so I'll merge after discussion.